README.md

```diff
@@ -86,6 +86,28 @@
    inv load-data
    ```
 
+1. _Optional_ Start Flower monitoring tool
+
+     by choosing a specific node to monitor
+
+     ```
+     inv start-flower --node <NODE-NAME>
+     ```
+
+     or start a separate flower instance for all of the nodes with
+
+     ```
+     inv start-flower --all
+     ```
+
+     Then go to the respective address on your browser to start monitoring the nodes.
+
+1. _Optional_ Kill all flower instances at any point with
+
+     ```
+     inv kill-flower
+     ```
+
```

Comment:
Does moving the whole flower documentation outside of the `Local Deployment` make sense?

It is something outside of the main deployment process and could have it's own "header". We ideally want simple and minimum setup instructions.

Not sure though, have a look and see if it makes it clearer.

You can add it below like the `Local Deployment (without single configuration file)`.